spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractMessageChannel.java

```diff
@@ -243,7 +244,7 @@ public void setMessageConverter(MessageConverter messageConverter) {
 	 * Return a read-only list of the configured interceptors.
 	 */
 	@Override
-	public List<ChannelInterceptor> getChannelInterceptors() {
+	public List<ChannelInterceptor> getInterceptors() {
```

Comment:
Can't do this; breaking change. Need to deprecate the old method.

Reply:
Unfortunately it clashes with the `InterceptableChannel` API. So, we in chicken-or-egg situation anyway, independently of the version we are going to fix it. Good deal that this method is just `protected`, so visible only for implementors, but not regular end-user code

Reply:
I've moved it to 5.1.4; let's talk when we have more time.